### PR TITLE
Ft help center #147010569

### PR DESCRIPTION
**Pull request by bmwachajr**
**What does this PR do?**
THis Pull request adds a help center with a FAQ that will ease a users process of getting onboard and help answer some common questions that may arise.

**Description of task completed?**
GIVEN I am a new user to the platform and I am interested in getting onboard
WHEN I look for the documentation on the site concerning how to start and find my way on the platform
THEN I should find a myriad of tutorials, help videos, live chat, FAQ's etc that will ease my process of getting onboard and help answer some common questions that may arise.

**What is the relevant pivotal tracker stories?**
pivotal [](https://www.pivotaltracker.com/story/show/147010569)

screen shot 2017-07-11 at 17 35 09
screen shot 2017-07-11 at 17 35 21

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,10 +8,13 @@ static-collected
 
 .DS_Store
 static/.DS_Store
+<<<<<<< HEAD
```
Comment (andela-landia) [inline .gitignore]: remove git conflict heads

File: .gitignore [hunk 2]
```diff
@@ -7,10 +8,13 @@ static-collected
...
+=======
 hc-venv
 hc/local_settings_old.py
 hc/local_urls.py
 local_settings.py
 hc-env
 .vscode/
 .idea/
+>>>>>>> develop
```
Comment (andela-landia) [inline .gitignore]: remove git conflict marks

File: .idea/claymore-healthchecks.iml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (andela-landia) [inline .idea/claymore-healthchecks.iml]: add  this file to gitignore

File: hc/accounts/tests/test_profile.py [hunk 4]
```diff
@@ -18,12 +18,17 @@ def test_it_sends_set_password_link(self):
         self.alice.profile.refresh_from_db()
         token = self.alice.profile.token
         ### Assert that the token is set
+<<<<<<< HEAD
```
Comment (andela-landia) [inline hc/accounts/tests/test_profile.py]: remove conflict heads

File: hc/accounts/tests/test_switch_team.py [hunk 5]
```diff
@@ -14,20 +14,30 @@ def test_it_switches(self):
         r = self.client.get(url, follow=True)
 
         ### Assert the contents of r
+<<<<<<< HEAD
```
Comment (andela-landia) [inline hc/accounts/tests/test_switch_team.py]: remove git confict errors

File: hc/api/models.py [hunk 6]
```diff
@@ -68,6 +70,7 @@ def log_url(self):
     def email(self):
         return "%s@%s" % (self.code, settings.PING_EMAIL_DOMAIN)
 
+    #use
```
Comment (domiebett) [inline hc/api/models.py]: What is this comment for??

**Review by andela-landia — CHANGES_REQUESTED** [review]
Add .idea folder to gitignore
Remove all git conflict heads ; clean up your code

File: hc/api/models.py [hunk 7]
```diff
@@ -189,6 +205,8 @@ def transport(self):
     def notify(self, check):
         # Make 3 attempts--
         for x in range(0, 3):
+
+            ###que wtf
```
Comment (domiebett) [inline hc/api/models.py]: And this?

File: .idea/claymore-healthchecks.iml (hunk 3 above)
Comment (bmwachajr) [reply .idea/claymore-healthchecks.iml]: Added to git ignore

**Review by andela-landia — COMMENTED** [review]
Remove the .idea folder from github using `git rm `

**Pull request merged**